azure-pipelines.yml

```
@@ -0,0 +1,34 @@
+# ASP.NET
+# Build and test ASP.NET projects.
+# Add steps that publish symbols, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
+
+trigger:
+- Assignment1
```

You probably want to have it match on more than one branch so that you can use it on future assignments.
```suggestion
- Assignment*
```